site/metrics.md

```diff
@@ -20,6 +20,15 @@ exposed:
 * `weave_dns_entries` - Number of DNS entries.
 * `weave_flows` - Number of FastDP flows.
 
+#### Publish Router Metrics Endpoint
+
+By default weave only listens on it's local interface to serve metrics. 
+To publish your metrics throughout your cluster, e.g. if
+your prometheus server is installed on a different host machine,
+you need to set `WEAVE_STATUS_ADDR` to listen on all interfaces:
+`WEAVE_STATUS_ADDR=0.0.0.0:6782`
```

Comment:
Also they may want to change the port number, if 6782 clashes with another port in use on their machine.